src/components/form/Form.styeld.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
+	background-color: #f0f0f0;
+	width: 300px;
+	height: 620px;
```

Comment:
Fixed heights often lead to problems in regards of responsiveness. Maybe you can find a different solution?

Reply:
is changed